src/app/store.ts

```diff
@@ -1,8 +1,15 @@
-import { combineSlices, configureStore } from '@reduxjs/toolkit';
+import { combineSlices, configureStore, Store } from '@reduxjs/toolkit';
+import { todosSlice } from '../features/todos';
+import { filterSlice } from '../features/filter';
+import { currentTodoSlice } from '../features/currentTodo';
 
-const rootReducer = combineSlices();
+const rootReducer = combineSlices({
```

Comment:
The function `combineSlices` is not a valid function in Redux Toolkit. You should use `combineReducers` instead to combine your slice reducers into a single root reducer.